src/edit_session.js

```diff
@@ -11,6 +11,7 @@ var Range = require("./range").Range;
 var Document = require("./document").Document;
 var BackgroundTokenizer = require("./background_tokenizer").BackgroundTokenizer;
 var SearchHighlight = require("./search_highlight").SearchHighlight;
+var TooltipMarkerGroup = require("./tooltip_marker").TooltipMarkerGroup;
```

Comment:
Would be better to call this simply markerGroup because we will use it for occurence highlights too which do not have a tooltip, having a tooltip is just an additional feature of this markers not the defining characteristic.

Reply:
Renamed in new commit